### PR TITLE
Minor style adjustments in dark mode

### PR DESCRIPTION
**Pull request by Auz**
Adjustments to borders and styles to look better in dark mode.

### CODE REVIEW

File: packages/front-end/components/MetricAnalysis/MetricAnalysis.tsx [hunk 1]
```diff
@@ -303,8 +307,8 @@ const MetricAnalysis: FC<MetricAnalysisProps> = ({
 
   const outdated = isOutdated(factMetric, metricAnalysis);
   return (
-    <div className="mb-4">
-      <div className="appbox p-3 mb-3">
+    <div className={`mb-4 ${outerClassName ? outerClassName : ""}`}>
+      <div className={`appbox p-3 mb-3 ${className}`}>
```
Comment (tzjames) [inline packages/front-end/components/MetricAnalysis/MetricAnalysis.tsx:311]: {$className || ""}

**Review by tzjames — APPROVED** [review]
Looks good besides those two minor nits.  

No screenshots or Test Plan?  
I assume you looked in both dark mode and light mode and it both looked good.

File: packages/front-end/styles/global.scss [hunk 2]
```diff
@@ -1374,7 +1396,7 @@ main.main.report {
 }
 
 .nav-link {
-  padding: 0.3rem 0.8rem;
+  padding: 0.3rem 1.5rem;
```
Comment (tzjames) [inline packages/front-end/styles/global.scss:1399]: Is this used in many places?  Does it look good in all of them?

Comment (Auz) [reply packages/front-end/styles/global.scss]: Yes

**Pull request merged**